app/src/main/java/net/gsantner/markor/activity/PreviewActivity.java

```diff
@@ -60,6 +61,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            WebView.enableSlowWholeDocumentDraw();
```

Comment:
can this be toggled before picture creation too?

Reply:
If I understood your question correctly, there is no need to toogle this configuration, because the [documentation](https://developer.android.com/reference/android/webkit/WebView.html#enableSlowWholeDocumentDraw()) says _"This method should be called before any WebViews are created."_ and there is no method like _"disableSlowWholeDocumentDraw()"_.

Reply:
does it make a big speed impact when using this in markor?

Reply:
No